samples/tab-mail/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-mail-ts",
+    "source": "officeDev",
+    "title": "Compose Mail in Outlook using Personal Tab app.",
+    "shortDescription": "This sample app shows how to compose mail in Outlook using personal tab app.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-mail/ts",
+    "longDescription": [
+      "This sample app shows how to compose mail in Outlook using personal tab app."
+    ],
+    "creationDateTime": "2023-06-19",
+    "updateDateTime": "2023-06-10",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "reactjs"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "reactjs"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "tab,mail,outlook"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/tab-mail/ts/Images/tab-mail.gif",
+        "alt": "Solution UX shows how to compose Mail in Outlook using Personal Tab app."
```

Comment:
Solution UX shows how to compose mail in Outlook using personal tab app.

Reply:
fixed